### PR TITLE
XIONE-5311 : Box stuck on Black screen after FSR

### PR DESCRIPTION
**Pull request by acheri988**
Reason for change:
Box stuck on Black screen after FSR
Test Procedure: None
Risks: Low

Change-Id: I03a6c9d2ce6a65ab79843b8a90173e036e13c443
Signed-off-by:Anooj Cheriyan <Anooj_Cheriyan@comcast.com>

### CODE REVIEW

File: HdmiCec_2/HdmiCec_2.cpp [hunk 1]
```diff
@@ -342,78 +343,105 @@ namespace WPEFramework
...
+
+               char c;
+               IARM_Result_t retVal = IARM_RESULT_SUCCESS;
+               retVal = IARM_Bus_Call_with_IPCTimeout(IARM_BUS_CECMGR_NAME, IARM_BUS_CECMGR_API_isAvailable, (void *)&c, sizeof(c), 1000);
+               if(retVal != IARM_RESULT_SUCCESS) {
+                   msg = "IARM_BUS_CECMGR is not available";
+                   LOGERR("CECMGR is not available. Failed to activate HdmiCec_2 Plugin");
+                   return msg;
+               } else {
+                   LOGERR("CECMGR is available. Activate HdmiCec_2 Plugin. IsCecMgrActivated: %d", IsCecMgrActivated);
```
Comment (anand-ky) [inline HdmiCec_2/HdmiCec_2.cpp:375]: Pls move this to LOGINFO

File: HdmiCec_2/HdmiCec_2.cpp [hunk 2]
```diff
@@ -342,78 +343,105 @@ namespace WPEFramework
...
+                   fflush (stdout);
```
Comment (anand-ky) [inline HdmiCec_2/HdmiCec_2.cpp:376]: Pls remove these flush statements.

File: HdmiCec_2/HdmiCec_2.cpp [hunk 3]
```diff
@@ -558,7 +587,9 @@ namespace WPEFramework
        {
             if(true == getEnabled())
             {
+                LOGWARN("%s %d. Calling setEnabled false", __func__, __LINE__); fflush (stdout);
```
Comment (anand-ky) [inline HdmiCec_2/HdmiCec_2.cpp:590]: Pls remove fflush

**Review by anand-ky — CHANGES_REQUESTED** [review]

**Comment by acheri988** [comment]
Review comment updated. New PR: https://github.com/rdkcentral/rdkservices/pull/1668

**Pull request closed without merging**